content/nim/monitoring/security-monitoring/configure/create-role-security-monitoring.md

```diff
@@ -1,17 +1,15 @@
 ---
 title: Add user access to Security Monitoring dashboards
-description: Learn how to grant users access to the F5 NGINX Security Monitoring dashboards.
-toc: true
 weight: 200
+toc: true
 doctype: how-to
 product: NIM
 docs: DOCS-1026
-
 ---
 
 ## Overview
 
-You can use F5 NGINX Security Monitoring to monitor NGINX App Protect WAF instances. The Security Monitoring analytics dashboards and security logs provide protection insights and help you analyze possible threats or identify opportunities to tune your security policies.
+You can use the Security Monitoring module to monitor NGINX App Protect WAF instances. The Security Monitoring analytics dashboards and security logs provide protection insights and help you analyze possible threats or identify opportunities to tune your security policies.
```

Comment:
Coming in NIM 2.19, Sec Mon will be a built-in feature of NIM and not a standalone module.